Classes/Settings/SettingsViewController.swift

```diff
@@ -84,8 +84,8 @@ GitHubSessionListener {
             case .success(let response):
                 let text: String
                 let color: UIColor
-                switch response.data.status {
-                case .good:
+                switch response.data.status.indicator {
+                case .none:
                     text = NSLocalizedString("Good", comment: "")
```

Comment:
Do you want to use the provided `description` key here instead?

Reply:
I've changed it to the `description` JSON property:

![image](https://user-images.githubusercontent.com/1848357/49872092-4f6cca00-fe18-11e8-9af8-75b6f6ce24c4.png)

Reply:
How does this look on iPhone 5SE?